core/templates/dev/head/pages/topic_editor/subtopics_editor/subtopic_editor_modal_directive.html

```diff
@@ -6,57 +6,54 @@ <h3>
   </div>
   <div class="modal-body">
     <div class="subtopic-title">
-      <span ng-if="!subtopicTitleEditorIsShown" ng-click="openSubtopicTitleEditor()"
-            class="oppia-editable-section">
-        <strong style="font-size: 1.2em;"><[editableTitle]></strong>
-        <i class="material-icons oppia-editor-edit-icon"
-           title="Edit Subtopic Title">&#xE254;
-        </i>
+      <p>
+        <strong style="font-size: 1.2em;">Subtopic Title</strong>
+      </p>
+      <form class="form-horizontal" role="form">
+        <input class="form-control" type="text" ng-blur="updateSubtopicTitle(editableTitle)" ng-model="editableTitle" maxlength="80" ng-change="resetErrorMsg()">
+      </form>
+      <span class="help-block" style="font-size: smaller; color: red;" ng-if="errorMsg">
+        <em><[errorMsg]></em>
       </span>
-      <span ng-if="subtopicTitleEditorIsShown">
-        <form class="form-horizontal" role="form" ng-submit="updateSubtopicTitle(editableTitle)">
-          <input type="text" ng-model="editableTitle">
-          <button type="submit" ng-disabled="editableTitle === ''" class="btn btn-success btn-sm">Save</button>
-          <button class="btn btn-default btn-sm" ng-click="closeSubtopicTitleEditor()">Cancel</button>
-        </form>
+      <span class="help-block" style="font-size: smaller; color: red;" ng-if="editableTitle.length === 80">
+        <em>The maximum length of subtopic title is 80 characters.</em>
       </span>
     </div>
     <div class="subtopic-page-editor">
-      <strong> Subtopic Page </strong>
-      <div ng-if="!subtopicEditorIsShown" class="preview">
-        <angular-html-bind html-data="htmlData">
+      <p>
+        <strong style="font-size: 1.2em;">Subtopic Page</strong>
+      </p>
+      <div ng-if="!subtopicEditorIsShown" class="preview" ng-click="closePreviewSubtopicPage(htmlData)">
+        <span ng-if="htmlData === ''" class="oppia-placeholder">
+          Click here to add content for the subtopic pages.
```

Comment:
Click here to edit the page that explains this subtopic.

Reply:
Done.